<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditScoreWindow</class>
 <widget class="QMainWindow" name="EditScoreWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Score</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lnEditScoreShooterName">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>51</y>
      <width>286</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>51</y>
      <width>166</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtSelect">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>50</y>
      <width>89</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelectedShooter">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>95</y>
      <width>626</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>711</width>
      <height>376</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tblScores">
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>sid</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>event</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>division</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>cal</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>score</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ButtUpdate">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>520</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtCancel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>520</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string>CANCEL/CLOSE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSession">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>571</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>comboBoxDate</tabstop>
  <tabstop>lnEditScoreShooterName</tabstop>
  <tabstop>ButtSelect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
